src/binaryen-c.h

```diff
@@ -3626,6 +3626,13 @@ BINARYEN_API void BinaryenSetColorsEnabled(bool enabled);
 
 // Query whether color is enable for the Wasm printer
 BINARYEN_API bool BinaryenAreColorsEnabled();
+
+// Clear up all interned types (dangerous)
+BINARYEN_API void BinaryenClearInternedTypes();
+
+// Clear up all interned strings (dangerous)
+BINARYEN_API void BinaryenClearInternedStrings();
```

Comment:
Maybe the API names could be scarier, like the internal API names are (`destroyAllStringsForTestingPurposesOnly` etc.). Perhaps `BinaryenClearInternedStringsForTestingPurposesOnly` or `BinaryenDangerClearInternedStrings`..?